modules/spring-config/src/main/java/org/mule/runtime/config/spring/handlers/MuleNamespaceHandler.java

```diff
@@ -433,11 +431,6 @@ public void init() {
     // Life-cycle Adapters Factories
     registerBeanDefinitionParser("custom-lifecycle-adapter-factory", new ChildDefinitionParser("lifecycleAdapterFactory"));
 
-    // Stores
-    registerBeanDefinitionParser("in-memory-store", new ChildDefinitionParser("store", InMemoryObjectStore.class));
```

Comment:
Is InMemoryObjectStore the implementaiton used by ObjectStoreManager with persistent=false? if not delete it